assets/js/index.js

```diff
@@ -0,0 +1,458 @@
+//target start button
+const startButton = document.getElementById("start-btn");
+
+// target main section
+const main = document.getElementById("main");
+const timerSpan = document.getElementById("time-span");
+
+// current question index
+let questionIndex = 0;
+
+//current score index
+let scoreIndex = 0;
+
+//define time for quiz
+let timeCountdown = 5;
+
+//all questions array
+const questions = [
+  {
+    text: "who has the highest amount of goals in a premier league season?: ",
+    options: ["Mohammed Salah", "Alan Shearer", "Cristiano Ronaldo", "Suarez"],
+    answer: "Mohammed Salah",
+  },
+  {
+    text: "which club has the highest amount of premier league titles? ",
+    options: [
+      " Derby County",
+      "Liverpool",
+      "Manchester City",
+      "Manchester United",
+    ],
+    answer: "Manchester United",
+  },
+  {
+    text: "Who has the most assist in a premier league season: ",
+    options: ["Kevin de Britney", "Thierry Henry", "Ryan Giggs", "Mesut özil"],
+    answer: "Thierry Henry",
+  },
+  {
+    text: "How many premier league titles have Manchester city won? ",
+    options: ["5", "8", "2", "11"],
+    answer: "5",
+  },
+  {
+    text: "which club has won premier league with no losses? ",
+    options: ["chelsea", "leicester", "Arsenal", "Manchester City"],
+    answer: "Arsenal",
+  },
+  {
+    text: "How many managers have lost their jobs in the 2019/20 Premier League season?",
+    options: ["7", "1", "4", "6"],
+    answer: "6",
+  },
+  {
+    text: "Who got more assists? ",
+    options: ["Wayne Rooney", "Frank Lampard", "Matic", "Nicolas Anelka"],
+    answer: "Wayne Rooney",
+  },
+  {
+    text: "Which player has scored the most free-kicks? ",
+    options: ["David Beckham", "Ronaldo", "Messi", "Drogba"],
+    answer: "David Beckham",
+  },
+];
+// function to render banner
+const renderBanner = () => {
+  const section = document.createElement("section");
+  section.setAttribute(
+    "class",
+    " content-section animate__animated animate__bounceInDown"
+  );
+  section.setAttribute("id", "banner");
+  const h1 = document.createElement("h1");
+  h1.setAttribute("class", "content-section-title");
+  h1.textContent = "welcome to the hardest premier league quiz";
+  const h2 = document.createElement("h2");
+  h2.setAttribute("class", "content-section-info");
+  h2.textContent = "enter at your own risk";
+  const btnDiv = document.createElement("div");
+  btnDiv.setAttribute(
+    "class",
+    "btn-control animate__animated animate__bounceInDown"
+  );
+  const button = document.createElement("button");
+  button.setAttribute("class", "btn");
+  button.setAttribute("id", "start-btn");
+  button.textContent = "start";
+  btnDiv.append(button);
+  section.append(h1, h2, btnDiv);
+  main.append(section);
```

Comment:
Your highscores page isn't working as this line is erroring - this `main` element does not exist on that page so check how you are targetting it.